### DISCUSSION TITLE
I have not yet received the benefits of the GitHub student Developer Pack.

### DISCUSSION THREAD

**Discussion by arjunkhatipatel** [community/community#134492] [GitHub Education]
### Hi everyone,

My GitHub education application was approved but it's been 3 days and I still haven't received any mail regarding the benefits I'll be getting after the approval. It's still showing your benefits will be available soon.

**Comment by kings099** [comment]
same

**Reply by shweta0dixit** [reply]
same, 4 days have been passed but i didn't get access yet

**Reply by deep-sengupta** [reply]
Check your email (including spam), verify your application status on GitHub Education, and contact GitHub support if needed.
OR
Tweet - "Hi @GitHubEducation, it’s been 4 days since I applied for the GitHub Student Developer Pack and I haven’t received access yet. Could you please assist? Thanks!"

**Comment by deep-sengupta** [comment]
Generally, once your application is approved, you should receive an email with instructions on how to access your benefits within a few days. However, it can sometimes take up to a week or more due to high demand or processing delays.
If it has been more than a week since your application was approved and you still haven't received any communication regarding your benefits, it's a good idea to contact GitHub support for assistance. They can provide more specific information about your situation and help resolve any issues.

**Reply by arjunkhatipatel** [reply]
Unable to contact Github. In the contact form it does not even have option for my problem and when i selected the other most relevant option Status of GitHub Education, It says Check your application status over at GitHub Education with a link.

**Reply by deep-sengupta** [reply]
If the GitHub contact form doesn’t fit your issue, try these steps:

Check GitHub Support Documentation: Visit their [support docs](https://docs.github.com/en/github) for relevant information.
Use GitHub Community: Post your question on the [GitHub Community Forum](https://github.community/).
Reach Out on Twitter: Contact GitHub on [Twitter](https://twitter.com/github) for assistance.

You can tweet something like this:
@github I’m having trouble with [brief description of your issue]. The contact form options don’t fit my problem. Can you help or direct me to the right support? Thanks!

**Reply by arjunkhatipatel** [reply]
Just tweeted to github about same, Let's see if they response.

**Comment by shweta0dixit** [comment]
I've checked my email but found nothing. I've already tweeted about this
issue, so let's see when they respond. Just to confirm, should I be
checking my college email or personal email?

**Reply by arjunkhatipatel** [reply]
You should check your college email.

**Reply by deep-sengupta** [reply]
You should check the email address you used when applying for the GitHub Student Developer Pack. If you used your college email for the application, check there. If you used your personal email, then check that one.
